recipes/watset/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "2.8.2" %}
+
+package:
+  name: watset
+  version: {{ version }}
+
+source:
+  url: https://github.com/nlpub/watset-java/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 7a957f55f219fbda7c8102b27bb7d8037a0a61a70d9f63b90d7b4746fd35a64f
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - openjdk >=11
```

Comment:
@dustalov I missed that before but with this constraint, it will build with the latest openjdk available.
You might want to force building with openjdk 11 instead to ensure it can run on >=11?

Reply:
The code requires JDK ≥ 11 and it should build on the more recent versions. Building with the latest OpenJDK available is perfectly fine.

Reply:
OK, fine!